src/test/java/com/google/api/codegen/testdata/discoveries/go/go_adexchangebuyer.v1.4.json.baseline

```diff
@@ -141,10 +140,9 @@ func main() {
     log.Fatal(err)
   }
 
-  // TODO: Change placeholders below to desired parameter values for the `Patch` call:
-
   // The account id
   id := int64(0)
+  // TODO: Change placeholder value above.
```

Comment:
It was preferred to alternative approaches, including having the TODO above the line. We didn't consider having it below, but I suspect there's no difference.